src/cloud-element-templates/create/CalledDecisionBindingProvider.js

```diff
@@ -0,0 +1,28 @@
+import {
+  ensureExtension
+} from '../CreateHelper';
+import { getDefaultValue } from '../Helper';
+
+
+export class CalledDecisionBindingProvider {
+  static create(element, options) {
+    const {
+      property,
+      bpmnFactory
+    } = options;
+
+    const {
+      binding
+    } = property;
+
+    const {
+      property: propertyName
+    } = binding;
+
+    const value = getDefaultValue(property);
+
+    const calledDecision = ensureExtension(element, 'zeebe:CalledDecision', bpmnFactory);
```

Comment:
@yT0n1 what happens if there are two properties on that binding?

Reply:
`ensureExtension` returns the first one